LLK/scratch-www
# [Master] remove email opt-in from join flow

hotfix version of PR https://github.com/LLK/scratch-www/pull/4095 which resolves https://github.com/LLK/scratchr2/issues/5723

Test plan:

Already tested on staging:
1. create new account
2. confirm that opt in field does not appear in form
3. confirm that account creation works
4. confirm that email confirmation works
5. confirm that sharing a project works

For production, we should repeat these steps.

We should also confirm that the email address was not added to our Mailchimp list.
